pkg/virt-handler/device-manager/socket_device.go

```diff
@@ -118,7 +160,14 @@ func NewSocketDevicePlugin(socketName, socketDir, socket string, maxDevices int)
 			Health: pluginapi.Healthy,
 		})
 	}
-	return dpi
+	//	if err := dpi.setSocketDirectoryPermissions(); err != nil {
```

Comment:
Is this not needed?

Reply:
it is part of a try to check what it is going on the CI since unit tests fail in the CI but not locally. It is part of the remove commit